packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -145,6 +119,17 @@ Widget _wrapWithBackground({
   );
 }
 
+Widget _wrapActiveColor(Color color, BuildContext context, Widget child) {
```

Comment:
Can you add a dart doc about why this method exists and what it does?  I know it's private - this is for other framework devs.

Reply:
SG, added